packages/cli-repl/test/e2e.spec.ts

```diff
@@ -521,12 +521,20 @@ describe('e2e', function() {
     describe('log file', () => {
       it('creates a log file that keeps track of session events', async() => {
         await shell.executeLine('print(123 + 456)');
-        await eventually(() => {
+        await eventually(async() => {
           expect(shell.output).to.include('579');
+          const log = await readLogfile();
+          expect(log.filter(logEntry => /rewritten-async-input/.test(logEntry.msg)))
+            .to.have.lengthOf(1);
```

Comment:
This is a drive-by fix that should hopefully take care of [this flaky failure](https://evergreen.mongodb.com/task/mongosh_linux_test_m42x_n12_1c2642e3300bed1fbe8d18cf2961fe1ff8009eda_20_12_04_09_05_45):

```
[2020/12/04 09:20:05.868]   1 failing
[2020/12/04 09:20:05.868]   1) e2e
[2020/12/04 09:20:05.868]        config and logging
[2020/12/04 09:20:05.868]          log file
[2020/12/04 09:20:05.868]            creates a log file that keeps track of session events:
[2020/12/04 09:20:05.868]       AssertionError: expected [] to have a length of 1 but got 0
[2020/12/04 09:20:05.868]       + expected - actual
[2020/12/04 09:20:05.868]       -0
[2020/12/04 09:20:05.868]       +1
[2020/12/04 09:20:05.868]       at Context.<anonymous> (test/e2e.spec.ts:529:20)
```